<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registered</class>
 <widget class="QWidget" name="Registered">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="register_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>34</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="err_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>317</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>请将带*项填满！</string>
   </property>
  </widget>
  <widget class="QLabel" name="userid_star">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>49</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userid">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="job">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_star">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>126</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passwd_again_star">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>164</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwd_again">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="sex_man">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sex_women">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>47</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>163</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>357</y>
     <width>85</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>357</y>
     <width>85</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>87</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="userid_label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="username_label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名字：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="passwd_label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="passwd_again_label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sex_label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="age_label">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="job_lable">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>工作：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
